x-pack/plugins/spaces/public/spaces_manager/spaces_manager.ts

```diff
@@ -49,11 +55,11 @@ export class SpacesManager {
   public async getSpaces(options: GetAllSpacesOptions = {}): Promise<GetSpaceResult[]> {
     const { purpose, includeAuthorizedPurposes } = options;
     const query = { purpose, include_authorized_purposes: includeAuthorizedPurposes };
-    return await this.http.get('/api/spaces/space', { query });
+    return await this.http.get('/api/spaces/space', { query, version });
```

Comment:
I had to add the version option to the core http requests here for the versioned endpoints. The version option is still denoted as experimental in the comments - is using the header option preferred?

Reply:
Added/fixed comment in 262673dc0727550c3d7ad54e172da5f69b38bcee & c85715aa1c307922da9c693b5151558d4158f4c1